src/core/core.tooltip.js

```diff
@@ -810,9 +811,13 @@ var exports = Element.extend({
 			ctx.fillStyle = textColor;
 			helpers.each(bodyItem.before, fillLineOfText);
 
-			helpers.each(bodyItem.lines, function(line) {
+			len = bodyItem.lines.length;
+			for (i = 0; i < len; i++) {
+				var line = bodyItem.lines[i];
+				var firstLine = index === 0;
```

Comment:
I think this should be `i`? It doesn't look like `index` is ever set anywhere and should be renamed to `i`

Reply:
I wanted to maintain the `index` variable inside the loop body, but ok.

Reply:
Fixed - gulp test passes now